.travis.yml

```diff
@@ -9,7 +9,10 @@ cache:
 
 matrix:
     include:
-        - php: hhvm
+        - php: hhvm-3.18
+          sudo: required
+          dist: trusty
+          group: edge
```

Comment:
And as we are bumping version to 1.4, it looks like this is a great occasion to do so.